server/seed/index.js

```diff
@@ -0,0 +1,51 @@
+require("dotenv").config();
+const colors = require("colors");
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const { userArray, profileArray } = require("./sampleProfiles");
+const connectDB = require("../db");
+
+connectDB();
+
+console.log("loading mock users");
+userArray.forEach(async (user) => await User.create(user));
+
+async function createProfile() {
+  try {
+    const users = await User.find();
+    const userId = users.map((user) => user._id);
+    let index = 0;
+
+    for (let profile of profileArray) {
+      profile.userId = userId[index];
+      await Profile.create(profile);
+      index++;
+    }
+    console.log("User ID's: ", userId);
+    console.log("populating profiles...");
+
+    setTimeout(async () => {
+      const profiles = await Profile.find().populate("userId", {
+        username: 1,
+        email: 1,
+      });
+      console.log(profiles);
+      console.log(`
+      seeding complete 
+      summary: 
+        Number of users created : ${users.length}
+        Number of profiles created : ${profiles.length}
+        Exiting process
+      `);
```

Comment:
This portion is unsuitable here. Please remove.

Reply:
I don't understand what you mean by "portion". Please explain. 😕. Did you mean the console logs or the setTimeout?